biomage/rds/tunnel.py

```diff
@@ -0,0 +1,47 @@
+import pathlib
+from subprocess import run
+
+import click
+
+from ..utils.constants import STAGING
+
+
+@click.command()
+@click.option(
+    "-i",
+    "--input_env",
+    required=False,
+    default=STAGING,
+    show_default=True,
+    help="Input environment of the RDS server.",
+)
+@click.option(
+    "-t",
+    "--endpoint_type",
+    required=False,
+    default="reader",
+    show_default=True,
+    help="The type of the rds endpoint you want to connect to, can be either reader or writer",
```

Comment:
Considering that the reader endpoint type actually points to the writer in our case, I'm concerned that someone might assume that if he's using the reader endpoint, it's safe leading to data loss and errors.  Should we leave it while we develop until we create some roles? I'm ok with merging this but I'm concerned about it misleading devs once we have migrated and are developing normally

Reply:
Based on conversations over slack I'll disable this parameter